snuba/web/rpc/v1/endpoint_time_series.py

```diff
@@ -160,8 +171,55 @@ def _convert_result_timeseries(
             if not row_data:
                 timeseries.data_points.append(DataPoint(data=0, data_present=False))
             else:
+                upper_confidence_limit = None
+                average_sample_rate = 0
+                sample_count = None
+                for col_name, col_value in row_data.items():
+                    if (
+                        col_name not in group_by_labels
+                        and col_name not in aggregation_labels
+                        and col_name != "time"
+                    ):
+                        custom_column_information = get_custom_column_information(
+                            col_name
+                        )
+                        if (
+                            custom_column_information.referenced_column is None
+                            or custom_column_information.referenced_column
+                            != timeseries.label
+                        ):
+                            continue
+
+                        if custom_column_information.column_type == "upper_confidence":
+                            upper_confidence_limit = col_value
+                        elif (
+                            custom_column_information.column_type
+                            == "average_sample_rate"
+                        ):
+                            average_sample_rate = col_value
+                        elif custom_column_information.column_type == "count":
+                            sample_count = col_value
+
+                reliability = Reliability.RELIABILITY_UNSPECIFIED
+                if upper_confidence_limit is not None and sample_count is not None:
+                    is_reliable = calculate_reliability(
+                        row_data[timeseries.label],
+                        upper_confidence_limit,
+                        sample_count,
+                    )
+                    reliability = (
+                        Reliability.RELIABILITY_HIGH
+                        if is_reliable
+                        else Reliability.RELIABILITY_LOW
+                    )
+
```

Comment:
this should be tested separately

Reply:
in a test: "here's what the row looks like, I am calculating the reliability of that row in this way"